script/releases/v1.0.0-slashing/cleanup/script.go

```diff
@@ -0,0 +1,323 @@
+package main
+
+import (
+	"context"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	proofgen "github.com/Layr-Labs/eigenpod-proofs-generation/cli/core"
+	eth2client "github.com/attestantio/go-eth2-client"
+	"github.com/attestantio/go-eth2-client/api"
+	v1 "github.com/attestantio/go-eth2-client/api/v1"
+	attestantio "github.com/attestantio/go-eth2-client/http"
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	multicall "github.com/jbrower95/multicall-go"
+	"github.com/samber/lo"
+)
+
+type EigenpodInfo struct {
+	Address                    string `json:"address"`
+	CurrentCheckpointTimestamp uint64 `json:"currentCheckpointTimestamp"`
+}
+
+type TQueryAllEigenpodsOnNetworkArgs struct {
+	Ctx               context.Context
+	AllValidators     []ValidatorWithIndex
+	Eth               *ethclient.Client
+	EigenpodAbi       abi.ABI
+	PodManagerAbi     abi.ABI
+	PodManagerAddress string
+	Mc                *multicall.MulticallClient
+}
+
+//go:embed EigenPod.abi.json
+var EigenPodAbi string
+
+//go:embed EigenPodManager.abi.json
+var EigenPodManagerAbi string
+
+type ValidatorWithIndex struct {
+	Validator *v1.Validator
+	Index     phase0.ValidatorIndex
+}
+
+type TArgs struct {
+	Node       string
+	BeaconNode string
+	Sender     string
+}
+
+func main() {
+	err := runScript(TArgs{
+		Node:       os.Getenv("RPC_URL"),
+		BeaconNode: os.Getenv("BEACON_URL"),
+		Sender:     os.Getenv("SENDER_PK"),
+	})
+	if err != nil {
+		fmt.Printf("Error: %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func panicOnError(msg string, err error) {
+	if err != nil {
+		fmt.Printf("Error: %s", msg)
+		panic(err)
+	}
+}
+
+func runScript(args TArgs) error {
```

Comment:
For validation, I assume we plan to run this script twice - once to complete checkpoints, again to validate none are pending?